tools/clang_tidy.py

```diff
@@ -8,28 +8,35 @@
 import sys
 
 
-DEFAULT_FILE_PATTERN = r".*\.[ch](pp)?"
+DEFAULT_FILE_PATTERN = r".*\.c(c|pp)?"
```

Comment:
Err.. you're no longer picking up headers now. Is that intentional?

Reply:
Yes clang tidy cannot process headers because headers are not compiled